doc/api/links.md

```diff
@@ -0,0 +1,76 @@
+Link Relations
+==============
+
+Link relations describe how two documents relate to each other.
+
+It is important to navigate through the links.  Do not remember nor code against URIs because they are subject to change at any time.  Always follow `Link` headers.
```

Comment:
I believe this is an improper use of "nor". "Do not" could be replaced with "Neither" to make it work.